app/views/pages/style.html.erb

```diff
@@ -1,9 +1,90 @@
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/default.min.css">
+<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>
+<script>hljs.initHighlightingOnLoad();</script>
+
 <div class="text-center">
```

Comment:
Small nit -- can you place this below the additions you've made? I want to keep the original theme styles and our custom components separate.